docs/products/access/api-reference/config-api/identity-provider-api.md

```diff
@@ -93,13 +94,104 @@ Pragma: no-cache
+| scopes                       | TULIP, OAUTH | no       | Space-separated scopes.                                                                                                                                                                                                 |
+| end_session_enabled          | TULIP        | no       | Specify whether End Session integration is enabled for this Identity Provider. <br/> Default value: `false`.                                                                                                            |
+| integrations                 | TULIP        | no       | List of enabled integrations. <br/> Supported values: `APP_TO_WEB`, `UDH_API`.                                                                                                                                          |
+| tulip_api_client_id          | TULIP        | depends  | Client identifier for Tulip API calls. <br/> Required when `APP_TO_WEB` or `UDH_API` integration is enabled.                                                                                                            |
+| tulip_api_client_secret      | TULIP        | depends  | Client secret for Tulip API calls. <br/> Required when `APP_TO_WEB` or `UDH_API` integration is enabled and authentication method is `client_secret_basic` or `client_secret_post`.                                     |
+| tulip_api_base_url           | TULIP        | depends  | This should be the base url of the Tulip brand without a trailing slash. UDH and App To Web will use this as a base for their urls. <br/> Required when `APP_TO_WEB` or `UDH_API` integration is enabled.               |
+| tulip_api_access_scope       | TULIP        | depends  | Space-separated scopes for the required Tulip segments e.g. `iwelcome:segment:example`. <br/> Required when `APP_TO_WEB` or `UDH_API` integration is enabled.                                                           |
+| authorization_url            | OAUTH        | yes      | Oauth authorization endpoint.                                                                                                                                                                                           |
+| token_url                    | OAUTH        | yes      | Oauth token endpoint.                                                                                                                                                                                                   |
+| profile_url                  | OAUTH        | yes      | OpenID Connect UserInfo endpoint.                                                                                                                                                                                       |
```

Comment:
I believe that this field is either named wrongly or it is marked as required due to some historical reasons.

Reply:
It's definitely named wrongly.
I described it in details here: https://github.com/onewelcome/access/pull/1629#discussion_r1254191390